zipkin-autoconfigure/storage-elasticsearch/src/main/java/zipkin/autoconfigure/storage/elasticsearch/ZipkinElasticsearchStorageProperties.java

```diff
@@ -36,6 +36,24 @@
    */
   private String index = "zipkin";
 
+
+  /**
+   * Affects scalability by limiting the number of nodes in the Elasticsearch cluster that can
+   * write to the daily index. Defaults to 5.
+   *
+   * <p>Corresponds to <a href="https://www.elastic.co/guide/en/elasticsearch/reference/current/index-modules.html">index.number_of_shards</a>
+   */
+  private int indexShards = 5;
+
+  /**
+   * Affects scalability by increasing the number of nodes in the Elasticsearch cluster that can
```

Comment:
```The number of replica copies of each shard in the index. Each shard and its replicas are assigned to a machine in the cluster. Increasing the number of replicas and machines in the cluster will improve read performance, but not write performance. Number of replicas can be changed for existing indices. Defaults to 1. It is highly discouraged to set this to 0 as it would mean a machine failure results in data loss.```